internal/appconfig/setters.go

```diff
@@ -247,3 +247,15 @@ func (c *Config) SetVolumes(volumes []Volume) {
 		}
 	}
 }
+
+func (c *Config) Volumes() []Volume {
```

Comment:
totally a nit pick but this is not a setter, we have been placing v1/v2 compatibility getters in config.go mainly.

Reply:
that's what I get for relying on IDE features and not using my eyeballs. thanks!